examples/dt/nova/nova-three-cells/control-plane/service-values.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  preserveJobs: false
+  galera:
+    templates:
+      openstack:
+        storageClass: lvms-local-storage
+        storageRequest: 5G
+        secret: osp-secret
+        replicas: 3
+      openstack-cell1:
+        storageClass: lvms-local-storage
+        storageRequest: 5G
+        secret: osp-secret
+        replicas: 3
+      openstack-cell2:
+        storageClass: lvms-local-storage
+        storageRequest: 5G
+        secret: osp-secret
+        replicas: 3
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:swift
+      [glance_store]
+      default_backend = default_backend
+      [default_backend]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+    default:
+      replicas: 1
+
+  swift:
+    enabled: true
+  telemetry:
+    enabled: true
+    template:
+      ceilometer:
+        enabled: true
+
+  rabbitmq:
+    templates:
+      rabbitmq-cell2:
+        replicas: 3
+
+  nova:
+    template:
+      secret: osp-secret
+      cellTemplates:
+        cell0:
+          cellDatabaseAccount: nova-cell0
+          cellDatabaseInstance: openstack
+          cellMessageBusInstance: rabbitmq
+          conductorServiceTemplate:
+            replicas: 3
+          hasAPIAccess: true
+        cell1:
+          cellDatabaseAccount: nova-cell1
+          cellDatabaseInstance: openstack-cell1
+          cellMessageBusInstance: rabbitmq-cell1
+          conductorServiceTemplate:
+            replicas: 3
+          hasAPIAccess: true
+        cell2:
+          cellDatabaseAccount: nova-cell2
+          cellDatabaseInstance: openstack-cell2
+          cellMessageBusInstance: rabbitmq-cell2
+          conductorServiceTemplate:
+            replicas: 3
+          hasAPIAccess: true
```

Comment:
Just a note for the future. We can spice this setup up by defining cell2 without api DB access (i.e. no upcall support).